corehq/apps/domain/models.py

```diff
@@ -713,6 +713,7 @@ def save_copy(self, new_domain_name=None, new_hr_name=None, user=None,
             new_domain.internal = InternalProperties()
             new_domain.creating_user = user.username if user else None
             new_domain.date_created = datetime.utcnow()
+            new_domain.use_sql_backend = True
```

Comment:
How was this being set before?

Reply:
It wasn't, which makes me thing it was just being copied over https://github.com/dimagi/commcare-hq/blob/4b4e8d102cbcd3cc68732f1e2fa41a10ac28df70/corehq/apps/domain/models.py#L692

Reply:
hmm. maybe we should run manual migration to set https://github.com/dimagi/commcare-hq/blob/4b4e8d102cbcd3cc68732f1e2fa41a10ac28df70/corehq/apps/domain/models.py#L272 and change the default property to `True`

Reply:
>run manual migration

How do you mean?

Reply:
Offlined and this is probably unreasonable